dsHost.c

```diff
@@ -58,6 +64,15 @@ dsError_t dsGetCPUTemperature(float *cpuTemperature)
     char Temp_File[FILE_SIZE];
     char temp_value[SIZE];
     FILE *fp = NULL;
+    if (!host_initialized) {                                                                                                                                               
+    dsError_t ret = dsERR_NOT_INITIALIZED;                                                                                                                          
```

Comment:
Please remove unnecessary declaration; you can return the error code directly right ?

Reply:
removed unnecessary declaration